### DISCUSSION TITLE
xDrip+ sync followers lose data when in standby

### DISCUSSION THREAD

**Discussion by PhilTU** [NightscoutFoundation/xDrip#3588] [Bug reports]
Following on from https://github.com/NightscoutFoundation/xDrip/issues/3582 multiple users of multiple phone brands and models report losing the data reception and hence alarms etc when the screen is off / phone in standby mode with screen locked.

Reports via Facebook group. Users say they updated master and follower(s).

Some say it isn't a problem for them.

**Comment by Kustomwireless** [comment]
after doing updates and restarting all phones   We are having issues where 2 followers are not getting a reading for 15 mins at a time and then  with out doing anything it starts working again the master phone is a Samsung a15 and followers are A Motorola 5g the other is a OnePlus. It has happened 3 times this morning we are in the US. Good luck Hope this is helpfull!
